<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1500</width>
    <height>840</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Swan</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="mainLayout" rowstretch="0,0,0,0,0,0" columnstretch="0,0">
    <item row="0" column="0">
     <widget class="QGraphicsView" name="gvCamera">
      <property name="verticalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOff</enum>
      </property>
      <property name="horizontalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOff</enum>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QGraphicsView" name="gvSensor">
      <property name="verticalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOff</enum>
      </property>
      <property name="horizontalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOff</enum>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="lblCameraXY">
      <property name="text">
       <string>x:--,y:--</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLabel" name="lblSensorXY">
      <property name="text">
       <string>x:--,y:--</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
    <item row="5" column="0" colspan="2">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QGroupBox" name="gboxScan">
        <property name="title">
         <string>扫描方式</string>
        </property>
        <layout class="QGridLayout" name="gridLayout">
         <item row="0" column="0">
          <widget class="QRadioButton" name="radScanByRect">
           <property name="text">
            <string>按范围扫描</string>
           </property>
           <property name="checked">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QRadioButton" name="radScanByDeg">
           <property name="text">
            <string>按角度扫描</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="1" column="3">
          <widget class="QLabel" name="label_10">
           <property name="text">
            <string>角度步长</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QSpinBox" name="txtRectStart">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="readOnly">
            <bool>true</bool>
           </property>
           <property name="maximum">
            <number>1920</number>
           </property>
          </widget>
         </item>
         <item row="0" column="4">
          <widget class="QSpinBox" name="txtRectStep">
           <property name="minimum">
            <number>10</number>
           </property>
           <property name="maximum">
            <number>1000</number>
           </property>
           <property name="singleStep">
            <number>10</number>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QSpinBox" name="txtDegStart">
           <property name="minimum">
            <number>-30</number>
           </property>
           <property name="maximum">
            <number>30</number>
           </property>
          </widget>
         </item>
         <item row="0" column="3">
          <widget class="QLabel" name="label_13">
           <property name="text">
            <string>像素步长</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item row="1" column="2">
          <widget class="QSpinBox" name="txtDegEnd">
           <property name="minimum">
            <number>-30</number>
           </property>
           <property name="maximum">
            <number>30</number>
           </property>
          </widget>
         </item>
         <item row="0" column="2">
          <widget class="QSpinBox" name="txtRectEnd">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="readOnly">
            <bool>true</bool>
           </property>
           <property name="maximum">
            <number>1920</number>
           </property>
          </widget>
         </item>
         <item row="1" column="4">
          <widget class="QDoubleSpinBox" name="txtDegStep">
           <property name="minimum">
            <double>0.100000000000000</double>
           </property>
           <property name="maximum">
            <double>60.000000000000000</double>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QRadioButton" name="radScanByPixel">
           <property name="text">
            <string>按像素扫描</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QSpinBox" name="txtPixStart">
           <property name="maximum">
            <number>1920</number>
           </property>
          </widget>
         </item>
         <item row="2" column="2">
          <widget class="QSpinBox" name="txtPixEnd">
           <property name="maximum">
            <number>1920</number>
           </property>
          </widget>
         </item>
         <item row="2" column="3">
          <widget class="QLabel" name="label_11">
           <property name="text">
            <string>像素步长</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item row="2" column="4">
          <widget class="QSpinBox" name="txtPixStep">
           <property name="minimum">
            <number>10</number>
           </property>
           <property name="maximum">
            <number>1000</number>
           </property>
           <property name="singleStep">
            <number>10</number>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="gboxline">
        <property name="title">
         <string>辅助线设定</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_5">
         <item row="0" column="0">
          <widget class="QLabel" name="label_15">
           <property name="text">
            <string>水平基准线</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item row="3" column="1">
          <widget class="QPushButton" name="btnPickColor">
           <property name="text">
            <string/>
           </property>
           <property name="flat">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="label_16">
           <property name="text">
            <string>基准中线</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QSpinBox" name="txtBasey1">
           <property name="minimum">
            <number>0</number>
           </property>
           <property name="maximum">
            <number>1080</number>
           </property>
           <property name="value">
            <number>360</number>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QSpinBox" name="txtBasex">
           <property name="minimum">
            <number>0</number>
           </property>
           <property name="maximum">
            <number>1920</number>
           </property>
           <property name="value">
            <number>960</number>
           </property>
          </widget>
         </item>
         <item row="3" column="0">
          <widget class="QLabel" name="label_17">
           <property name="text">
            <string>基准线色</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label_14">
           <property name="text">
            <string>基准线下</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QSpinBox" name="txtBasey2">
           <property name="minimum">
            <number>0</number>
           </property>
           <property name="maximum">
            <number>1080</number>
           </property>
           <property name="value">
            <number>720</number>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="gboxOp">
        <property name="title">
         <string>云台操作</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_4">
         <item row="2" column="1">
          <widget class="QPushButton" name="btnHome">
           <property name="text">
            <string>复位</string>
           </property>
          </widget>
         </item>
         <item row="1" column="2">
          <widget class="QPushButton" name="btnTurnRight">
           <property name="text">
            <string>向右设定原点↷</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QPushButton" name="btnTurnLeft">
           <property name="text">
            <string>↶向左设定原点</string>
           </property>
          </widget>
         </item>
         <item row="2" column="2">
          <widget class="QPushButton" name="btnCheckBarrier">
           <property name="text">
            <string>寻找物理原点</string>
           </property>
          </widget>
         </item>
         <item row="0" column="2">
          <widget class="QPushButton" name="btnPause">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>暂停扫描</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QPushButton" name="btnStartStop">
           <property name="text">
            <string>开始扫描</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="gboxSensor">
        <property name="title">
         <string>相机</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_6">
         <item row="2" column="1">
          <widget class="QPushButton" name="btnSensorVideo">
           <property name="text">
            <string>工业相机拍摄</string>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QLabel" name="label_12">
           <property name="text">
            <string>曝光时间（毫秒）</string>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QPushButton" name="btnSensorPreview">
           <property name="text">
            <string>工业相机拍照</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QSpinBox" name="txtExpo">
           <property name="minimum">
            <number>1</number>
           </property>
           <property name="maximum">
            <number>10000000</number>
           </property>
           <property name="singleStep">
            <number>500</number>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QCheckBox" name="chkMirrorH">
           <property name="text">
            <string>水平翻转</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QCheckBox" name="chkMirrorV">
           <property name="text">
            <string>垂直翻转</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1500</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
